catalog/ec2-policies.yaml

```diff
@@ -58,33 +58,289 @@
   resources:
     - "arn:aws:ec2:*:*:instance/*"
 
-# This policy denies instance types that aren't based on the Nitro system and don't support Encryption-in-Transit as
+# This policy denies instance types that do not support Encryption-in-Transit as
 # described in the following document:
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/data-protection.html#encryption-transit
+# and enumerated by the following command (specified in the documentation):
+#
+#  aws ec2 describe-instance-types \
+#  --filters Name=network-info.encryption-in-transit-supported,Values=true \
+#  --query "InstanceTypes[*].[InstanceType]" --output text | sort
 - sid: "DenyEC2InstancesWithoutEncryptionInTransit"
   effect: "Deny"
   actions:
     - "ec2:RunInstances"
   condition:
-    - test: "StringNotLike"
+    - test: "StringNotEquals"
       variable: "ec2:InstanceType"
       values:
-        - c5a.*
-        - c5ad.*
-        - c5n.*
-        - c6gn.*
-        - d3.*
-        - d3en.*
-        - g4ad.*
-        - g4dn.*
-        - i3en.*
-        - m5dn.*
-        - m5n.*
-        - m5zn.*
-        - p3dn.*
-        - p4d.*
-        - r5dn.*
-        - r5n.*
+      # updated 2022-09-26
+      - c5a.12xlarge
+      - c5a.16xlarge
+      - c5a.24xlarge
+      - c5a.2xlarge
+      - c5a.4xlarge
+      - c5a.8xlarge
+      - c5a.large
+      - c5a.xlarge
```

Comment:
Is there a reason you updated these to the specific instance types rather than the wildcard?